packages/edit-site/src/components/global-styles/font-library-modal/local-fonts.js

```diff
@@ -31,11 +32,11 @@ function LocalFonts() {
 	const { installFont } = useContext( FontLibraryContext );
 	const [ notice, setNotice ] = useState( null );
 	const [ isUploading, setIsUploading ] = useState( false );
-	const supportedFormats =
-		ALLOWED_FILE_EXTENSIONS.slice( 0, -1 )
-			.map( ( extension ) => `.${ extension }` )
-			.join( ', ' ) +
-		` ${ __( 'and' ) } .${ ALLOWED_FILE_EXTENSIONS.slice( -1 ) }`;
+	const { listItemSeparator } = l10n;
+
+	const supportedFormats = ALLOWED_FILE_EXTENSIONS.map(
+		( extension ) => `.${ extension }`
+	).join( listItemSeparator );
```

Comment:
This seems like an overkill because this info is unlikely to change.

Reply:
Glad to simplify all of that and just provide a string, agree it's a bit overkill.